python/Makefile

```diff
@@ -0,0 +1,87 @@
+#
+# This Makefile serves as convenient command-line auto-completion when working on Python bindings
+#
+#
+
+define default-help
+# invoke: 'make uninstall', 'make install'
+endef
+.PHONY: default
+default: build
+	@echo "## py: make default"
+	@echo "## py: make default [DONE]"
+
+
+define all-help
+# Do the common task during development of: uninstall clean build install test
+endef
+.PHONY: all
+all: uninstall clean build install
+
+define build-help
+# Generate cython pxd, ctypes, patch them, then create a source package
```

Comment:
This seems a bit misleading, the source package doesn't seem to be created in this target

Reply:
Updated